app/models/miq_product_feature/seeding.rb

```diff
@@ -0,0 +1,30 @@
+class MiqProductFeature < ApplicationRecord
+  module Seeding
+    extend ActiveSupport::Concern
+
+    RELATIVE_FIXTURE_PATH = "db/fixtures/miq_product_features".freeze
+    FIXTURE_PATH = Rails.root.join(RELATIVE_FIXTURE_PATH).freeze
+
+    module ClassMethods
+      def seed_files
+        return seed_root_filename, seed_core_files + seed_plugin_files
```

Comment:
As discussed, this return value feels strange to me, but I understand why it's done this way.  Maybe in a future PR we can pull the `everything` feature out of the root file and remove the special handling of that file.

Reply:
I like this idea, and definitely think we should do it in a followup.